task/sensor-polling/src/main.rs

```diff
@@ -0,0 +1,186 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Temperature and fan monitoring loop, for systems without thermal control
+
+#![no_std]
+#![no_main]
+
+use drv_i2c_devices::mwocp68::{Error as Mwocp68Error, Mwocp68};
+use ringbuf::*;
+use task_sensor_api::{Sensor, SensorError, SensorId};
+use userlib::*;
+
+task_slot!(I2C, i2c_driver);
+task_slot!(SENSOR, sensor);
+
+/// Type containing all of our temperature sensor types, so we can store them
+/// generically in an array.  Right now, we only support
```

Comment:
Are we missing the end of this comment? (If not and this was intentional, maybe add a `:` to the end?)

Reply:
Not intentional; fixed!